openmetadata-ui/src/main/resources/ui/src/components/Database/DatabaseSchema/DatabaseSchemaTable/DatabaseSchemaTable.tsx

```diff
@@ -160,6 +184,95 @@ export const DatabaseSchemaTable = ({
     }
   };
 
+  const handleDisplayNameUpdate = useCallback(
+    async (data: EntityName, id?: string) => {
+      try {
+        const schemaDetails = schemas.find((schema) => schema.id === id);
+        if (!schemaDetails) {
+          throw new Error(t('error.tableNotFound'));
+        }
+        const updatedData = { ...schemaDetails, displayName: data.displayName };
+        const jsonPatch = compare(schemaDetails, updatedData);
+        await patchDatabaseSchemaDetails(schemaDetails.id, jsonPatch);
+        setSchemas((prevData) =>
+          prevData.map((schema) =>
+            schema.id === id
+              ? { ...schema, displayName: data.displayName }
+              : schema
+          )
+        );
+      } catch (error) {
+        showErrorToast(error as AxiosError);
+      }
+    },
+    [schemas, t]
```

Comment:
No need to add `t` import in the dependency.

Reply:
Removed `t` from dependency